src/Games/NexusMods.Games.StardewValley/Emitters/MissingSMAPIEmitter.cs

```diff
@@ -0,0 +1,27 @@
+using NexusMods.Abstractions.Diagnostics;
+using NexusMods.Abstractions.Diagnostics.Emitters;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Games.StardewValley.Models;
+
+namespace NexusMods.Games.StardewValley.Emitters;
+
+public class MissingSMAPIEmitter : ILoadoutDiagnosticEmitter
+{
+    private static readonly Uri NexusModsSMAPIUri = new("https://nexusmods.com/stardewvalley/mods/2400");
+
+    public async IAsyncEnumerable<Diagnostic> Diagnose(Loadout loadout)
+    {
+        await Task.Yield();
```

Comment:
I looked up `Task.Yield()` a bit, for what purpose are you using it here?

Reply:
The return value of the method is `IAsyncEnumerable<T>` which is a bit clunky to work with when the entire method is synchronous.